server/services/userService.test.ts

```diff
@@ -2,32 +2,36 @@ import UserService from './userService'
 import HmppsAuthClient, { User } from '../data/hmppsAuthClient'
 import { UserClient } from '../data'
 
-import { paginatedResponseFactory, userFactory } from '../testutils/factories'
+import { paginatedResponseFactory, referenceDataFactory, userFactory } from '../testutils/factories'
 import { PaginatedResponse } from '../@types/ui'
 import { ApprovedPremisesUser } from '../@types/shared'
+import ReferenceDataClient from '../data/referenceDataClient'
 
 jest.mock('../data/hmppsAuthClient')
 jest.mock('../data/userClient')
+jest.mock('../data/referenceDataClient.ts')
 
 const token = 'some token'
 
 describe('User service', () => {
   const userClient: jest.Mocked<UserClient> = new UserClient(null) as jest.Mocked<UserClient>
   const userClientFactory = jest.fn()
   const userProfile = userFactory.build({ roles: ['workflow_manager', 'assessor'] })
+  const referenceDataClient = new ReferenceDataClient(null) as jest.Mocked<ReferenceDataClient>
 
   let hmppsAuthClient: jest.Mocked<HmppsAuthClient>
   let userService: UserService
 
   beforeEach(() => {
     jest.resetAllMocks()
-
+    const ReferenceDataClientFactory = jest.fn()
```

Comment:
I think it would be easier to scan if we mocked the clients in a uniform way. Either assign the `userClientFactory` in the `beforeEach` or assign the `referenceDataClientFactory` next to where the `referenceDataClient` is defined. 
Also please can we use camelCase for variable names